tag-security-privacy.md

```diff
@@ -0,0 +1,65 @@
+# TAG Security & Privacy Review Document
+
+Answering questions from [the questionaire doc here](https://w3ctag.github.io/security-questionnaire/).
+
+## Questions to Consider
+
+### 3.1. Does this specification deal with personally-identifiable information?
+Not directly, only if Pan/Tilt is supported by the camera and the value of Pan/Tilt.
+
+### 3.2. Does this specification deal with high-value data?
+No.
+
+### 3.3 Does this specification introduce new state for an origin that persists across browsing sessions?
+No.
+
+### 3.4. Does this specification expose persistent, cross-origin state to the web?
+No.
+
+### 3.5. Does this specification expose any other data to an origin that it doesn’t currently have access to?
+No.
+
+### 3.6. Does this specification enable new script execution/loading mechanisms?
+No.
+
+### 3.7. Does this specification allow an origin access to a user’s location?
+No.
+
+### 3.8. Does this specification allow an origin access to sensors on a user’s device?
+No.
+
+### 3.9. Does this specification allow an origin access to aspects of a user’s local computing environment?
+No.
+
+### 3.10. Does this specification allow an origin access to other devices?
+No.
+
+### 3.11. Does this specification allow an origin some measure of control over a user agent’s native UI?
+No.
+
+### 3.12. Does this specification expose temporary identifiers to the web?
+No. [GETUSERMEDIA](https://www.w3.org/TR/mediacapture-streams/) might expose deviceID. This specification does not expose anything extra.
+
+### 3.13. Does this specification distinguish between behavior in first-party and third-party contexts?
+No.
+
+### 3.14. How should this specification work in the context of a user agent’s "incognito" mode?
+The same as outside of incognito.
+
+### 3.15. Does this specification persist data to a user’s local device?
+No.
```

Comment:
I guess you're finding all these questions because this spec never started its way to CR, so this questionnaire was never run through and there's no security and privacy section in the Spec. With that said, I could see a malicious app setting e.g. `pan`, `tilt` and `zoom` e.g. to minimally different values and scooping them later on as an active fingerprinting technique, WDYT?

Reply:
> there's no security and privacy section in the Spec

The privacy and security considerations discussed in [GETUSERMEDIA](https://www.w3.org/TR/mediacapture-streams/) apply to this extension specification.

> I could see a malicious app setting e.g. pan, tilt and zoom e.g. to minimally different values and scooping them later on as an active fingerprinting technique, 

I think yes, and of course it applies to other constrainsts also. Spec might also add a note about implementation that - Resetting to a default value, say 0 Pan, 0 Tilt after every session.